test/src/screens/crypto_detail_screen_test.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:van_quang_tinh/src/screens/crypto_detail_screen.dart';
+import 'package:van_quang_tinh/src/constants/constants.dart' as constants;
+
+void main() {
+  group('Crypto Detail Screen Tests', () {
+    var widget = const MaterialApp(home: CryptoDetailScreen());
+
+    testWidgets('Should render Appbar.',
```

Comment:
too many tests for one single test, we should split this up into smaller tests

Reply:
I fixed it.